### PR TITLE
vpc info module

### LINKED ISSUE #164: Add `vpc_info` module

### PR DESCRIPTION
**Pull request by Polina-Gubina**
close #164

### CODE REVIEW

File: plugins/modules/vpc_info.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+from ansible_collections.opentelekomcloud.cloud.plugins.module_utils.otc import OTCModule
+
+
+class VpcInfoModule(OTCModule):
+    argument_spec = dict(
+        name_or_id=dict(required=False),
+        project_id=dict(required=False),
+        limit=dict(required=False, type='int')
+    )
```
Comment (anton-sidelnikov) [inline plugins/modules/vpc_info.py:89]: project_id and limit not used in module, why?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: plugins/modules/vpc_info.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DOCUMENTATION = '''
+---
+module: vpc_info
+short_description: Get vpc info from OpenTelekomCloud
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.0.3"
```
Comment (anton-sidelnikov) [inline plugins/modules/vpc_info.py:19]: fix please

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: plugins/modules/vpc_info.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    enable_shared_snat:
+      description: Specifies whether the shared SNAT function is enabled. The value true\
+       indicates that the function is enabled, and the value false indicates that the function is not enabled.
+      type: bool
+'''
+
+EXAMPLES = '''
+# Get a vpc info.
+- vpc_info:
+    project_id: "39007a7e-ee4f-4d13-8283-b4da2e037c69"
```
Comment (anton-sidelnikov) [inline plugins/modules/vpc_info.py:69]: remove pls

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Review by enrrou — APPROVED** [review]

**Review by lego963 — DISMISSED** [review]

File: plugins/modules/vpc_info.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+options:
+  name_or_id:
+    description:
+      - Name or id of the vpc.
+    type: str
+requirements: ["openstacksdk", "otcextensions"]
+'''
+
+RETURN = '''
+vpc_info:
```
Comment (outcatcher) [inline plugins/modules/vpc_info.py:32]: Actual return is `vpcs`:
```python
        self.exit_json(
            changed=False,
            vpcs=data
        )
```

File: plugins/modules/vpc_info.py [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+DOCUMENTATION = '''
+---
+module: vpc_info
+short_description: Get vpc info from OpenTelekomCloud
+extends_documentation_fragment: opentelekomcloud.cloud.otc
+version_added: "0.11.1"
+author: "Polina Gubina(@polina-gubina)"
+description:
+  - Get vpc backend server group from the OTC.
```
Comment (outcatcher) [inline plugins/modules/vpc_info.py:22]: It's not a backend server group

**Review by outcatcher — CHANGES_REQUESTED** [review]

**Review by outcatcher — APPROVED** [review]

**Pull request merged**